bndtools.core/src/bndtools/views/repository/RepositoriesView.java

```diff
@@ -464,6 +466,17 @@ public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
                 return Status.OK_STATUS;
             }
         }.schedule();
+        KeyAdapter adapter = new KeyAdapter() {
```

Comment:
Refresh is one of those globally recognized key sequences in Eclipse.  So in this view we can register an action (that is a button on view toolbar) that hooks into the global 'refresh' action.  That way if the user customizes their refresh action globally, this will pick up on those changes instead of hard-coding to 'F5'.  I will try to find an example of this and send you the source.